data/Searson_2004/metadata.yml

```diff
@@ -0,0 +1,266 @@
+source:
+  primary:
+    key: Searson_2004
+    bibtype: Article
+    year: '2004'
+    author: M. J. Searson and D. S. Thomas and K. D. Montagu and J. P. Conroy
+    journal: Tree Physiology
+    title: Wood density and anatomy of water-limited eucalypts
+    volume: '24'
+    number: '11'
+    pages: 1295--1302
+    doi: 10.1093/treephys/24.11.1295
+contributors:
+  data_collectors:
+  - last_name: Searson
+    given_name: Matthew J.
+    affiliation: Centre for Horticulture and Plant Sciences, University of Western
+      Sydney
+    ORCID: .na
+    additional_role: no contact
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code: '
+    data %>%
+      mutate(
+        location = "Hawkesbury campus",
+        vessel_diameter = 2*sqrt(`Vessel lumen area (um2 )`)/3.14159
```

Comment:
I think pi also has to be in the square root, like: 
```2*sqrt(`Vessel lumen area (um2 )`/3.14159)```

Reply:
Fixed but please check if it's correct

Reply:
Yes, you're right, I had brackets in the wrong place